src/main/java/types/CpuPinningPolicy.java

```diff
@@ -69,6 +69,17 @@
      * @status added
      * @since 4.5
      */
-    RESIZE_AND_PIN_NUMA;
+    RESIZE_AND_PIN_NUMA,
+
+    /**
+     * The CPU pinning will be automatically calculated by the engine when a vm starts and it will be dropped
+     * as soon as the vm stops.
```

Comment:
Would it be better to add that the CPUs used will be exclusively used by the VM?

Reply:
What about: The CPU pinning will be automatically calculated by the engine when a vm starts and it will be dropped when the vm stops. The pinning is exclusive, that means that no other VM can use the pinned pCPU.

Reply:
Sounds perfect to me.